cosmos-server/src/it/scala/com/mesosphere/cosmos/test/CosmosIntegrationTestClient.scala

```
@@ -113,9 +123,13 @@ object CosmosIntegrationTestClient extends Matchers {
       requestBody: Req,
       requestMediaType: MediaType,
       responseMediaType: MediaType,
-      status: Status = Status.Ok
+      status: Status = Status.Ok,
+      method: String = "POST"
     )(implicit decoder: Decoder[Res], encoder: Encoder[Req]): Xor[ErrorResponse, Res] = {
-      val request = buildPost(path, requestBody, requestMediaType, responseMediaType)
+      val request =
+        if(method == "POST") buildPost(path, requestBody, requestMediaType, responseMediaType)
+        else buildGet(path, responseMediaType)
```

As I said earlier, I'm okay with using a `String` for the HTTP method for now, but let's throw an error if `method` is not `POST` or `GET`.